tests/util/test_default.py

```diff
@@ -521,3 +524,15 @@ def test_entry_point_relative_error_if_doesnt_exist(tmp_directory):
 ])
 def test_extract_name(arg, expected):
     assert default.extract_name(arg) == expected
+
+
+def test_mixed_envs(tmp_directory):
+    # Set a conda env with an install
+    _write_sample_conda_env()
+    process = subprocess.Popen(
+        "conda install pygraphviz -c conda-forge -y".split(),
+        stdout=subprocess.PIPE)
```

Comment:
this line will take a ~1min to run, it's better to just fake the file and add the `pypgraphviz @ file:` thing